src/asic/config.py

```diff
@@ -122,7 +124,11 @@ def load_asic_file_config() -> dict[str, ASICFileConfig]:
     """Return a list of ASIC file configurations"""
     # This is a stream-like object. If you want the actual info, call
     # stream.read()
-    resource = importlib.resources.files("asic").joinpath("data/ASIC_FILE_CONFIG.jsonl")
+    path_env = os.getenv("PATH_ASIC_FILE_CONFIG")
+    if path_env is None:
+        raise ValueError("PATH_ASIC_FILE_CONFIG environment variable not set")
```

Comment:
point to an example public file in the error so the user can learn how to create the `ASIC_FILE_CONFIG`

Reply:
Done